src/ImageManipulation.php

```diff
@@ -782,6 +789,31 @@ public function getHeight()
         return 0;
     }
 
+    /**
+     * Return whether image will be lazy loaded
+     *
+     * @return bool
+     */
+    public function IsLazyLoaded()
+    {
+        if (!Config::inst()->get('SilverStripe\\Assets\\Image', 'lazy_loading_enabled')) {
+            return false;
+        }
+        return $this->lazyLoad;
+    }
+
+    /**
+     * Set whether image will be lazy loaded
+     *
+     * @param bool $lazyLoad
+     * @return $this
+     */
+    public function LazyLoad($lazyLoad)
+    {
+        $this->lazyLoad = $lazyLoad;
```

Comment:
Need to check that width and height exist on the image

If not, return false

Will need to unit test this